lib/model/binding.dart

```diff
@@ -195,16 +234,6 @@ class LiveZulipBinding extends ZulipBinding {
     return url_launcher.closeInAppWebView();
   }
 
-  @override
-  Future<BaseDeviceInfo> deviceInfo() async {
-    final deviceInfo = await device_info_plus.DeviceInfoPlugin().deviceInfo;
-    return switch (deviceInfo) {
-      device_info_plus.AndroidDeviceInfo(:var version)   => AndroidDeviceInfo(sdkInt: version.sdkInt),
-      device_info_plus.IosDeviceInfo(:var systemVersion) => IosDeviceInfo(systemVersion: systemVersion),
-      _                                                  => throw UnimplementedError(),
-    };
-  }
-
```

Comment:
Ah, and I didn't notice last time: is there a reason to move this `device_info_plus`-related code before other things in the class? I think `ZulipBinding`, `LiveZulipBinding`, and `TestZulipBinding` should declare their members in the same order; the consistency should help reading and editing them together.

Reply:
Same question :slightly_smiling_face: